index.bs

```diff
@@ -16,13 +16,439 @@ WPT Display: closed
+"<code>[=storage access status/active=]</code>". In order to support forward-compatibility with as-yet-unknown
+semantics, servers SHOULD ignore this header if it contains an invalid value.
+
+<pre class="example" id="sec-fetch-storage-access-usage">
+
+// When the request's credentials mode is "`omit`", the header is omitted:
+
+
+// When the request is same-site, the header is omitted:
+
+
+// When the request has no access to cross-site cookies, the header is "<code>[=storage access status/none=]</code>":
+Sec-Fetch-Storage-Access: none
+
+// When the request has no access to cross-site cookies, but
+// 'storage-access' permission has already been granted, the header is
+// "<code>[=storage access status/inactive=]</code>":
+Sec-Fetch-Storage-Access: inactive
+
+// When the request has access to cross-site cookies, the header is "<code>[=storage access status/active=]</code>":
+Sec-Fetch-Storage-Access: active
+
+</pre>
+
+<div algorithm>
+    To <dfn export lt="set-storage-access">set the `Sec-Fetch-Storage-Access` header</dfn> for a [=request=] |request|:
+
+    <ol class="algorithm">
+        1.  Assert: |request|'s [=request/url=] is a [=potentially trustworthy URL=].
+        1. If |request|'s [=request/credentials mode=] [=string/is=] not "`include`", abort these steps.
+        1.  Let |access| be |request|'s [=request/storage access status=].
+        1.  If |access| is null, abort these steps.
+        1.  Let |value| be a [=Structured Field=] value whose value is a [=structured field/token=].
+        1.  Set |value|'s value to |access|.
+        1.  [=header list/Set a structured field value=] given ("`Sec-Fetch-Storage-Access`", |value|) in |request|'s [=request/header list=].
+    </ol>
+</div>
+
+
+The `Activate-Storage-Access` HTTP Response Header {#activate-storage-access-header}
+-------------------------------------------------------------------------------------
+
+The <dfn http-header export>`Activate-Storage-Access`</dfn> HTTP response header
+allows a server to opt in to accessing its cookies in a cross-site request
+context. It is a [=Structured Field=] [=structured field/item=] whose value MUST be a [=structured
+field/token=]. [[!I-D.structured-field-values-for-http]] Its ABNF is:
+
+```
+Activate-Storage-Access = sf-item
+```
+
+Valid `Activate-Storage-Access` values include "`load`" and "`retry`". In order
+to support forward-compatibility with as-yet-unknown semantics, servers SHOULD
+ignore this header if it contains an invalid value.
+
+The following parameter is defined:
+
+* A parameter whose key is "`allowed-origin`", and whose value is a [=structured field/string=]. See below for processing requirements.
+
+<pre class="example" id="activate-storage-access-usage">
+
+// The server's response requests that the user agent activate storage access
+// before continuing with the load of the resource.
+Activate-Storage-Access: load
```

Comment:
I wonder if we should mention that this only makes sense in the context of document loads

Reply:
Done

Reply:
It is not clear to me why this is only relevant "when loading a new document". Could you maybe provide some more documentation somewhere in this PR why that's the case?